designsafe/static/scripts/data-depot/components/projects/curation-directory/curation-directory.component.js

```diff
@@ -206,14 +206,37 @@ class CurationDirectoryCtrl {
         });
     }
 
-  onBrowse(file) {
-    if (file.type === 'dir') {
-        this.$state.go(this.$state.current.name, {filePath: file.path.replace(/^\/+/, ''), query_string: null})
+    onBrowse(file) {
+        if (file.type === 'dir') {
+            this.$state.go(this.$state.current.name, {filePath: file.path.replace(/^\/+/, ''), query_string: null})
+        }
+        else {
+            this.FileOperationService.openPreviewModal({api: 'agave', scheme: 'private', file})
+        }
     }
-    else {
-        this.FileOperationService.openPreviewModal({api: 'agave', scheme: 'private', file})
+
+    filteredHazmapperMaps(maps) {
+        maps = maps ? maps : [];
+
+        maps.forEach(map => {
+            switch(map.deployment) {
+                case 'production':
+                    map.href = `https://hazmapper.tacc.utexas.edu/hazmapper/project/${map.uuid}`;
```

Comment:
what will we do about public maps?  I'm wondering if a user publishes a project, do the map owners need to make sure they allow public access (via make public map).  I'm not certain 🤔   We could just put some thoughts in a ticket for follow up later.

Reply:
@nathanfranklin Would it be best to have a separate case for that? Or could it be handled on the hazmapper side? Maybe by redirecting to the public map (if the user does not have access && there is a public map available)?